### ISSUE TITLE
Assignment 4 Questions Confusion

### ISSUE THREAD

**Issue by shahtrupt** [neuniversity/ALY6140#18]
Dear Professor and classmates,

I am confused about what exactly is expected from the below two questions mentioned in our Week 4 Assignment:
**1. Generate a box plot that shows average bill amount by day of week -** Do we need to create a plot which has 4 box plots by days of week showing their respective bill amounts? 
**2. Generate a box plot that shows average bill amount by gender -**  Do we need to create a plot which has 2 box plots by gender showing their respective bill amounts? 

Please let me know how should I interpret the above questions. Thanks in advance!

**Comment by Jin-pengSong** [comment]
I think we can create a plot which includes 4 box plots by days of week. 

By the way, I have a question about the statistics: Take the first row as an instance, whether the total_bill belongs to one consumer or two consumers? Whether the tip belongs to one or two consumers?

**Comment by shahtrupt** [comment]
I think we do not have to consider the size of the party here. I tried creating the plot that shows average bill amount by day of week. I am not sure whether this is what is expected.
![image](https://user-images.githubusercontent.com/44628610/49308031-e25a4b80-f48b-11e8-9510-5f40c603d0cc.png)

**Comment by Jin-pengSong** [comment]
I have the similar plot which looks like you.

**Comment by Jin-pengSong** [comment]
Do we need to consider the size in the above questions, such as the average
bill?

**Comment by shahtrupt** [comment]
I don't think so, as both the questions did not mention anything about the 'size' of the party in the average bill.

**Comment by Jin-pengSong** [comment]
Do you know how to show two plots side by side?

**Comment by CHENGYULIU1** [comment]
I generated the similar plot too!!

**Comment by Jin-pengSong** [comment]
I have already solved the side by side problem. However, still confuse how to use Bokeh to create boxplot

**Comment by pr24** [comment]
I generated same boxplot.

**Comment by echolq012** [comment]
I also have the problem with " average bill amount". Does anyone figure out the average bill amount?

**Comment by hansu101** [comment]
Hi guys,
I have similar box plots with you. But I have a question about showing side by side, can we control the location by plt.subplot ( , , )?

**Comment by CanlinJiang** [comment]
I used the plt.subplot() for this question, but also generate the above one...
